modules/geo-layers/src/tile-layer/tile-layer.js

```diff
@@ -82,14 +82,14 @@ export default class TileLayer extends CompositeLayer {
   }
 
   _getTilesetOptions(props) {
-    const maxZoom = Number.isFinite(this.state.maxZoom) ? this.state.maxZoom : props.maxZoom;
```

Comment:
@Pessimistress do you see something wrong with this? I've tested with mvtlayer and it works.

Reply:
Perfect, I think this is the right thing to do (TileLayer should not reference a state that itself doesn't define).